src/server/sockets.ts

```diff
@@ -1,11 +1,32 @@
 import { io } from './configs/socket.config.js';
+import { generateTournamentPin } from './helpers/socket.helper.js';
+
+const tournamentPool: any = {
+  // tPin: [
+  //     socketIds,
```

Comment:
Be careful here.. i know i had mine like this earlier .. but i changed it to Players .. you might want an array a Players so that there is more information

Reply:
Will refactor